src/components/ClientInactiveProjectsList.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { useQuery } from '@apollo/client'
+import Grid from '@material-ui/core/Grid'
+import { orderBy } from 'lodash'
+
+import LoadingProgress from './LoadingProgress'
+import ProjectsList from './ProjectsList'
+import ProjectsEmptyState from './ProjectsEmptyState'
+import { GET_CLIENT_INFO } from '../operations/queries/ClientQueries'
+
+const ClientInactiveProjectLists = ({
+    clientId,
+    history
+}) => {
+
+    const { loading, error, data } = useQuery(GET_CLIENT_INFO, {
+        fetchPolicy: 'cache-and-network',
+        variables: { id: Number(clientId) }
+    })
+
+    if (loading) return <LoadingProgress/>
+    if (error) return `Error! ${error.message}`
+
+    const getInactiveProjects = () => {
+        const inactiveProjects = []
+        data.getClientById.projects.map(p => {
+            if (!p.is_active) {
+                inactiveProjects.push(p)
+            }
+        })
+        return inactiveProjects
+    }
+    const projects = getInactiveProjects()
```

Comment:
How often this function gets called? and why not place it as a component state?

Reply:
The function gets called once per render.
I tried using a component state but it has problems when verifying the length inside the render.

Reply:
Suggestion with this pattern
```
const getInactiveProjects = () => {
    return data
        ? data.getClientById.projects.filter(p => !p.is_active)
        : []
}

const inactiveProjects = getInactiveProjects()

```